implementations/go/main.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/LordRahl90/botDevelopment/handlers"
+	"github.com/gorilla/mux"
+)
+
+func main() {
+	router := mux.NewRouter()
+
+	router.HandleFunc("/", handlers.HomeHandler).Methods("GET")
+	router.HandleFunc("/webhook", handlers.WebHookHandler).Methods("GET")
+
+	fmt.Println("Server Starting up")
+	log.Fatal(http.ListenAndServe(":3000", router))
```

Comment:
🎉 🎊 🎉

Reply:
Seems u r a genius gopher Lord, pls do well to direct me to any resource that could help me soul. Thanks for the feedback. its most appreciated... esp for a beginner like me....